src/ast/mod.rs

```diff
@@ -3131,12 +3141,38 @@ impl fmt::Display for Statement {
                 Ok(())
             }
             Statement::Truncate {
-                table_name,
+                table_name: _,
+                table_names,
                 partitions,
                 table,
+                only,
+                identity,
+                cascade,
             } => {
                 let table = if *table { "TABLE " } else { "" };
-                write!(f, "TRUNCATE {table}{table_name}")?;
+                let only = if *only { "ONLY " } else { "" };
+
+                let table_names = table_names
+                    .iter()
+                    .map(|table_name| table_name.to_string())
+                    .collect::<Vec<String>>()
+                    .join(", ");
+
+                write!(f, "TRUNCATE {table}{only}{table_names}")?;
```

Comment:
```suggestion
                write!(f, "TRUNCATE {table}{only}{table_names}", table_names = display_comma_separated(...))?;
```
does the helper function apply to this case, it might simplify the join if so?